.github/workflows/scripts/python/changelog_overview.py

```
@@ -0,0 +1,113 @@
+        logging.info(f"Metadata - Version: {version}, Title: {name}, Date: {date_str}")
+
+        entry = f"""        {{
+            Version = "{version}",
+            Name = "{name}",
+            Date = "{date_str}",
+            URL = "http://faforever.github.io/fa/changelog/{version}",
+            Path = "/lua/ui/lobby/changelog/generated/{markdown_file.stem}.lua"
+        }},"""
+        return entry, parsed_date
+
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)